addons/account/views/account_unit_test.xml

```diff
@@ -1,62 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I could leave it, but the file is not loaded anywhere, so just sowing confusion.